lib/smart_answer_flows/register-a-birth/register_a_birth.govspeak.erb

```diff
@@ -20,5 +20,7 @@
 
   You can still [apply for a UK passport for your child](/apply-renew-passport) even if you do not register the birth in the UK.
 
+^If you're a member of HM Armed Forces or you work with them, you may be able to register a birth abroad. Email <dbs-jcccgroupmailbox@mod.gov.uk> to check if you're eligible.^
```

Comment:
```suggestion
^If you’re a member of HM Armed Forces or you work with them, you may be able to register a birth abroad. Email <dbs-jcccgroupmailbox@mod.gov.uk> to check if you’re eligible.^
```

Reply:
Hi @issyl0 , thank you for spotting that. Can you please make the change?

Reply:
Hey both, I've made the changes requested.